README.md

```
@@ -232,114 +232,51 @@ Most `.ts` files in the inventory directories are **auto-generated** from upstre
-### Release Process
-
-1. **Update the Version**
-
-Edit the `version` field in `package.json` to the new version number, following [Semantic Versioning](https://semver.org/) (see [Version Management](#version-management) above).
-
-Then set the `VERSION` env var from `package.json` for use in subsequent steps:
-
-```bash
-export VERSION=$(cat package.json | jq .version -r)
-echo "Releasing version: v${VERSION}"
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)